addons/hr_timesheet/static/src/services/timesheet_uom_service.js

```diff
@@ -48,6 +48,10 @@ export const timesheetUOMService = {
         if (!registry.category("formatters").contains("timesheet_uom")) {
             registry.category("formatters").add("timesheet_uom", service.formatter);
         }
+        if (!registry.category("formatters").contains("timesheet_uom_no_toggle")) {
+            registry.category("formatters").add("timesheet_uom_no_toggle", service.formatter);
+        }
+
```

Comment:
Since this changes is in your PR in 16, I guess you can remove it in that PR, since you will add it during the forward port. 🙂

Reply:
Done! Thanks :slightly_smiling_face: